packages/prisma-generator-proto/src/generate/hooks.ts

```diff
@@ -1,17 +1,62 @@
 import { DMMF } from "@prisma/generator-helper";
 
 function generateSingleHook(model: DMMF.Model): string {
+  const initialValue: { [key: string]: string | number } = {};
+  for (const field of model.fields) {
+    if (field.kind === "scalar" && field.name !== "id") {
```

Comment:
Let's use `field.isId` for this purpose. This is because a model's id might not be named "id".